mllm/engine/HpcThreadPool.cpp

```
@@ -0,0 +1,206 @@
+
+#if defined(__gnu__linux__)
+#include <pthread.h>
+#include <sched.h>
+#endif
+
+#include <cstdint>
+
+namespace mllm {
+
+#if defined(__gnu_linux__)
+static cpu_set_t getNumaAffinity(void) {
```

Clang (14.0.6)

[error] redundant void argument list in function definition

(modernize-redundant-void-arg,-warnings-as-errors)